src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Verbs.tt

```diff
@@ -252,3 +256,74 @@ foreach (XElement verb in commands.Descendants(XName.Get("Verb","")))
 <#
 }
 #>}
+<#+
+public class OptionData
+{
+    public OptionData(XElement element)
+    {
+        string elementName = element.Name.LocalName;
+        Name = element.Attribute(XName.Get("Name", ""))?.Value;
+        LongName = element.Attribute(XName.Get("LongName", ""))?.Value;
+        Help = element.Attribute(XName.Get("Help", ""))?.Value;
+        if (Help != null) { Help = "Strings." + Help; }
+        Shortcut = element.Attribute(XName.Get("Shortcut",""))?.Value;
+        IsRequired = element.Attribute(XName.Get("Required", ""))?.Value == "true";
+
```

Comment:
``` cs
MultipleValues = element.Attribute(XName.Get("MultipleValues", ""))?.Value == "true" ? "true" : "false";
```
Can we add line of code here to support multiple values in `argument` element?

Reply:
Why don't you do a separate PR for this MultipleValues feature that you want to add for arguments (and other improvements you decide to make). You can do that as part of your using the feature.
If I add it to this PR, the feature isn't even being used yet, or tested in any way.